lk2nd/device/dts/msm8952/msm8917-huawei-agassi.dts

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+
+/*
+ * This device is quirky. It excluded from generic build.
+ * To run lk2nd on this device you need to flash custom bootloader and unlock it.
+ * Read more https://wiki.postmarketos.org/wiki/Huawei_Mediapad_T3_10_(huawei-agassi)#Bootloader_unlocking
+ *
+ * Compile lk2nd with the following flags to run on this device
+ * LK2ND_ADTBS="msm8917-huawei-agassi.dtb"
+ * LK2ND_COMPATIBLE="huawei,agassi"
+ */
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8917 0>;
+	qcom,board-id = <QCOM_BOARD_ID(QRD, 1, 0) 0>;
```

Comment:
I assume it's skua?

Reply:
I think here the file name doesn't really matter since this is a dtb that will only be manually included for that one device. We can rename it later if we have more here